salt/states/boto_ec2.py

```diff
@@ -386,6 +492,8 @@ def eni_absent(
     else:
         if __opts__['test']:
             ret['comment'] = 'ENI is set to be deleted.'
+            if release_eip and 'allocationId' in r['result']:
```

Comment:
@ryan-lane wouldn't this be the only check needed for a test=True state run?  Or am I missing something?